.env

```diff
@@ -0,0 +1 @@
+SECRET_TOKEN_KEY=0d1ac44d5198236d6433b73a684aae1072d764ab16fc3e93401bfb64696d3ff77d469723d94f591ecbe8cba9fd299d8f462642f682429f6395cd203b66d62a4c
```

Comment:
You should gitignore the .env since it consist of secrets and credentials. It should not be pushed to code.

Go through the article once.
https://dev.to/robusgauli/5-things-i-wish-i-knew-before-adding-a-variable-in-env-4op3